### PR TITLE
[Help Text] concatenated help text patches 

### PR DESCRIPTION
**Pull request by llevitis**
This PR replaces the 30+ previous help text PRs that were conflicting with one another 👎
I'll make a second PR for next week that'll have help text in Markdown format for all the modules (that shall be functional for any module that gets modularized and can support Markdown).

### CODE REVIEW

**Comment by driusan** [comment]
@kongtiaowang can you test the patch?

The content is the same as the previous PRs there were merged into this one, so I think it should be pretty straight forward..

File: SQL/Archive/18.1/2017-10-20_all_help_text_patches.sql [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+INSERT INTO `help` (parentID, hash, topic, content, created, updated) VALUES ((SELECT helpID FROM (SELECT * FROM `help`) as temp WHERE hash=md5('survey_accounts')),md5('addSurvey'), 'Add Survey','To create a new survey form for a participant to complete at home, users must enter the DCCID and PSCID of the candidate, then they must select the Visit Label and Instrument name from the corresponding dropdown menus.\r\n\r\n<b>To E-mail a survey directly</b>\r\nIf you wish to send a form within an email to a participant directly, users must enter the e-mail address of the participant  twice, at which point the “Email Survey” button will become active and users may press this button to e-mail the survey.\r\n\r\nOnce the user has clicked “Email Survey”, a pop-up window will appear displaying a generic message that will be sent in the e-mail by default. At this stage, users may edit the message and customize it to fit their needs.','2017-08-24 00:00:00',NULL);
```
Comment (driusan) [inline SQL/Archive/18.1/2017-10-20_all_help_text_patches.sql]: Is this subselect going to work?

Shouldn't it just be ((SELECT helpID from `help` WHERE hash=md5[...etc]) or am I missing something?

File: SQL/Archive/18.1/2017-10-20_all_help_text_patches.sql [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
...CID_VisitLabel) in order to be uploaded and inserted into the database.  \r\nClick “Browse” to select the file to upload from the local computer.  Click “Submit” to launch the upload. The newly uploaded file will be displayed in the table at the bottom.  As the upload progresses, refreshing the page will show the current status in the “Progress” column.\r\n\r\nUse the Selection Filters to find past uploads, filtering by “CandID”, “PSCID”, or “Visit Label”. As filter criteria are added, the data table of results below will dynamically update.  Click “Clear Form” to reset all filters and search fields. \r\nClick on any row in the data table to view the output of the imaging insertion pipeline for that file, which will appear in the “Log Viewer” panel at top right.  In this panel, users have the option to display a “Detailed” view of all pipeline output, or “Summary” of key pipeline steps.  This log information is displayed in the grey panel below;  scroll down within the grey panel to view additional output. \r\n\r\nTo view all DICOM header fields and metadata, click on “View Details” under the “Tarchive Info” column to load the DICOM Archive module.', created = '2017-08-29 00:00:00'
```
Comment (MounaSafiHarab) [inline SQL/Archive/18.1/2017-10-20_all_help_text_patches.sql]: @llevitis 

minor comment, but the button is now called "Clear Filters" not "Clear Forms" in this sentence `Click “Clear Form” to reset all filters and search fields`.

File: SQL/Archive/18.1/2017-10-20_all_help_text_patches.sql [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
...ting QC review), or both. \r\nAfter specifying search criteria in the selection filters, click “Show Data” to display results in the data table below, with one row per relevant scanning session. Click “Clear Form” to reset all selection filters. \r\n\r\nSort the data table results by clicking on any column header.   The date of first acquisition of all scans for the timepoint is indicated under the “First Acq Date” column.  The “Last QC” column shows the date of the most recent Quality Control (QC) on any scan in the session. New scans awaiting Quality Control (QC) will be marked “NEW” (in red) under the “New Data” column. \r\n\r\nTo download, visualize, add QC annotation or view details of imaging files for a candidate-timepoint, select a dataset type the “Links” columns: “native”, “selected” (best scan quality), or “all types”.. "Native" will show only raw imaging datasets, "Selected" displays only scans flagged by QC as having the best scan quality, and "all types" displays all data, including analyzed outputs. By clicking on any of these links, the user will be able to see details for that specific candidate’s dataset from a given scan session.', created = '2017-08-29 00:00:00'
```
Comment (MounaSafiHarab) [inline SQL/Archive/18.1/2017-10-20_all_help_text_patches.sql]: The column is now called 'First Acquisition" instead of 'First Acq Date'

**Comment by ridz1208** [comment]
@llevitis is this still valid givewn that mardown has taken over loris help ??

**Comment by llevitis** [comment]
@ridz1208 Sorry I just saw these comments - no, I don't think this PR is necessary anymore.

**Comment by driusan** [comment]
closed as per @llevitis's comment

**Pull request closed without merging**